src/extension/onboarding/wallet-restore/index.tsx

```diff
@@ -7,7 +7,8 @@ import { MnemonicField } from './mnemonic-field';
 import OnboardingForm from '../onboarding-form';
 import { init } from '../../../domain/repository';
 import { validateMnemonic } from 'bip39';
-import type { RestorationJSON, RestorationJSONDictionary } from '../../../application/account';
+import type { RestorationJSONDictionary } from '../../../application/account';
+import { checkRestorationDictionary } from '../../../application/account';
```

Comment:
I personally prefer this format:
```suggestion
import { checkRestorationDictionary, type RestorationJSONDictionary } from '../../../application/account';
```

Reply:
let's keep import type for consistency with other files (or change all...)